hypha/static_src/src/app/src/redux/selectors/submissions.js

```diff
@@ -75,6 +75,27 @@ const getSubmissionReminders = createSelector(getCurrentSubmission, submission =
     return submission && submission.reminders || []
 })
 
+const getSubmissionMetaTerms = createSelector(
+    [getCurrentSubmission], submission => {
+        let metaTerms = []
+        if(submission && "metaTerms" in submission){
```

Comment:
Will there be a case where a submission does not have `metaTerms` property?

From the SubmissionDetailSerializer class it looks like it will be always available.

Reply:
Yeah, Initially submission list API will be called, only after the current submission(usually 1st submission in list) set, detail submission API will be called. So, it  is mandatory to check meta terms exist or not.

Reply:
Got it. Thanks for the explanation! I think we should add a short comment explaining this case for future reference.